src/Uno.UI/UI/Xaml/Controls/Primitives/ToggleButton.mux.cs

```diff
@@ -227,6 +227,12 @@ private void OnToggleImpl()
 		/// </summary>
 		private void OnIsCheckedChanged()
 		{
+			if (_suppressCheckedChanged) // Uno workaround
```

Comment:
Sorry, didn't realize at first, but could you also add the pooling issue link / #id to this and the other workaround comments? Just in case we are not able to address this right now, so we later know which parts of code can be removed